tests/ops/test_channel_ops.py

```diff
@@ -51,18 +51,6 @@ def test_kraus_matrices_sum_identity(self, ops, p, tol):
         Kraus_sum = np.einsum("ajk,ajl->kl", K_arr.conj(), K_arr)
```

Comment:
Could we add the line

```python
pytestmark = pytest.mark.usefixtures("tape_mode")
```

to the top of this file? This will cause all tests in this file to be run in both tape and non-tape mode (currently, the tests only execute in non-tape mode).